pretix_eth/templates/pretix_eth/pending.html

```diff
@@ -1,13 +1,13 @@
-{% load i18n %}
-{% load static %}
-<script type="text/javascript" src="{% static "pretix_eth/check.js" %}"></script>
+{% load i18n %} {% load static %}
 <p>
-    <strong>{% blocktrans %}Deposit {{ amount }} {{ coin }} into {{ payment_info }}{% endblocktrans %}</strong>
+  <strong>
+    {% blocktrans %} Deposit {{ amount }} {{ coin }} into {{ payment_info }}
```

Comment:
I think this should be `{% blocktrans trimmed %}` to ensure that modifying whitespace in the translation string doesn't cause any related translations not to be found.

Reply:
Done 👍